[2025-03-10 10:52:21,150] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: xkcd_batch_loader.store_xkcd_data 2025-03-10T10:51:22.682887+00:00 [queued]>
[2025-03-10 10:52:21,163] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: xkcd_batch_loader.store_xkcd_data 2025-03-10T10:51:22.682887+00:00 [queued]>
[2025-03-10 10:52:21,163] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2025-03-10 10:52:21,163] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2025-03-10 10:52:21,163] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2025-03-10 10:52:21,180] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_xkcd_data> on 2025-03-10T10:51:22.682887+00:00
[2025-03-10 10:52:21,182] {standard_task_runner.py:52} INFO - Started process 174 to run task
[2025-03-10 10:52:21,183] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'xkcd_batch_loader', 'store_xkcd_data', '2025-03-10T10:51:22.682887+00:00', '--job-id', '73', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/XKCD_batch/dag.py', '--cfg-path', '/tmp/tmpnwrxpp03', '--error-file', '/tmp/tmp1sos47iv']
[2025-03-10 10:52:21,185] {standard_task_runner.py:77} INFO - Job 73: Subtask store_xkcd_data
[2025-03-10 10:52:21,269] {logging_mixin.py:104} INFO - Running <TaskInstance: xkcd_batch_loader.store_xkcd_data 2025-03-10T10:51:22.682887+00:00 [running]> on host 4dcdde3a97c5
[2025-03-10 10:52:21,344] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xkcd_batch_loader
AIRFLOW_CTX_TASK_ID=store_xkcd_data
AIRFLOW_CTX_EXECUTION_DATE=2025-03-10T10:51:22.682887+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-10T10:51:22.682887+00:00
[2025-03-10 10:52:21,375] {base.py:78} INFO - Using connection to: id: ***_postgresql. Host: ***, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2025-03-10 10:52:21,419] {base.py:78} INFO - Using connection to: id: ***_postgresql. Host: ***, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2025-03-10 10:52:21,445] {python.py:151} INFO - Done. Returned value was: None
[2025-03-10 10:52:21,459] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=xkcd_batch_loader, task_id=store_xkcd_data, execution_date=20250310T105122, start_date=20250310T105221, end_date=20250310T105221
[2025-03-10 10:52:21,503] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-10 10:52:21,515] {local_task_job.py:149} INFO - Task exited with return code 0
